src/main/scala/beam/utils/data/synthpop/NewYorkScenarioFilter.scala

```diff
@@ -0,0 +1,241 @@
+package beam.utils.data.synthpop
+
+import java.io.{Closeable, File}
+
+import beam.utils.ProfilingUtils
+import beam.utils.scenario.generic.readers.{CsvHouseholdInfoReader, CsvPersonInfoReader, CsvPlanElementReader}
+import beam.utils.scenario.generic.writers.{CsvHouseholdInfoWriter, CsvPersonInfoWriter, CsvPlanElementWriter}
+import beam.utils.scenario.{HouseholdId, PersonId, PersonInfo, PlanElement}
+import com.typesafe.scalalogging.LazyLogging
+
+import scala.collection.mutable
+import scala.util.Try
+
+private case class InputPath(path: String) extends AnyVal
+private case class OutputPath(path: String) extends AnyVal
+
+object NewYorkScenarioFilter extends LazyLogging {
+
+  def main(args: Array[String]): Unit = {
+    val input = InputPath("C:/repos/beam/test/input/newyork/generic_scenario/10034k")
```

Comment:
Should this be a param also? This'll be the last I comment on absolute paths - it is a standing suggestion.